src/style.css

```
@@ -1 +1,124 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;600;700&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;600;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;600;700&display=swap'" (import-notation)

(import-notation)